common/djangoapps/microsite_configuration/admin.py

```diff
@@ -15,4 +15,24 @@ class Meta(object):  # pylint: disable=missing-docstring
         model = Microsite
 
 
+class MicrositeHistoryAdmin(admin.ModelAdmin):
```

Comment:
just now, I refreshed and saw the comment on the archive table. We currently keep one outside of edxapp, so I came to the same conclusion, that we needed one, but I did not want to keep adding models so that is why I placed it outside